management-api-agent/pom.xml

```diff
@@ -170,8 +170,8 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.5.1</version>
                 <configuration>
-                    <source>1.8</source>
-                    <target>1.8</target>
+                    <source>11</source>
+                    <target>11</target>
```

Comment:
Won't this cause a problem because we support Cassandra 3.11 and it only supports Java 8, doesn't it?

Reply:
yes, this causes problems and so we can't really use JDK11 for the project itself unfortunately